adlfs/spec.py

```diff
@@ -2146,7 +2147,7 @@ async def _async_initiate_upload(self, **kwargs):
 
     _initiate_upload = sync_wrapper(_async_initiate_upload)
 
-    def _get_chunks(self, data, chunk_size=1024**3):  # Keeping the chunk size as 1 GB
+    def _get_chunks(self, data, chunk_size):
```

Comment:
Looking through this more, it probably makes sense to just remove the `chunk_size` parameter from this helper method all together and just switch the sole reference to chunk_size with the instance property `self.blocksize`. Mainly, we are not really overriding this value now and the block size is already set in the constructor. So it makes it simpler.